### PR TITLE
Add everything again

### PR DESCRIPTION
**Pull request by oskojess**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  "jest": {
+    "preset": "jest-expo"
+  },
+  "dependencies": {
+    "expo": "^23.0.4",
+    "react": "16.0.0",
+    "react-native": "0.50.3",
+    "react-navigation": "^1.0.0-beta.21",
+    "styled-components": "^2.3.0",
+    "util": "*"
```
Comment (jgcmarins) [inline package.json]: remove this

File: screens/BookDetail.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { StyleSheet, Text, View, ScrollView, Button } from 'react-native';
+import styled from 'styled-components/native';
+const util = require('util');
```
Comment (jgcmarins) [inline screens/BookDetail.js]: also remove this

File: package.json (hunk 1 above)
Comment (oskojess) [reply package.json]: okay

File: screens/BookDetail.js (hunk 2 above)
Comment (oskojess) [reply screens/BookDetail.js]: I remembered that you we're going to make the procedure with me, in order to "clone" the code in the MAC.
also the update file it's not in my pc either. In the meantime, I'll code in the oldest file.

Comment (jgcmarins) [reply screens/BookDetail.js]: you can clone this repo and continue from this branch

Comment (jgcmarins) [reply screens/BookDetail.js]: GitHub stands for it.
`git clone git@github.com:oskojess/ChallengeEntria.git`
`git checkout start-again`
that's all \o/

Comment (oskojess) [reply screens/BookDetail.js]: Success!

Thanks <3

**Pull request closed without merging**